app/app.go

```diff
@@ -498,7 +523,7 @@ func NewStrideApp(
 		app.IBCKeeper.ChannelKeeper,
 		app.DistrKeeper,
 		app.BankKeeper,
-		app.RatelimitKeeper, // ICS4Wrapper
+		app.HooksICS4Wrapper, // ICS4Wrapper
```

Comment:
Why this change?

Reply:
Traffic up the stack is routed via the ICS4Wrappers that are wired on each keeper. The stack was previously 
```
channel
...
ratelimit
pfm
transfer
```
where the ICS4 routing was:
* transfer wires pfm 
* pfm wires rate limit
* rate limit wires channel

---

Now the stack is
```
channel
...
ratelimit
ibchooks [NEW]
pfm
transfer
```
So the routing needs to be:
* transfer wires pfm 
* pfm wires IBC hooks [NEW]
* IBC hooks wires rate limit [NEW]
* rate limit wires channel

Reply:
Ah yeah, this always confuses me. So the order of outbound packets depends on the Keeper definitions?

Reply:
yeah exactly. The part that I could use a second set of eyes on is usually, you just pass the keeper as the ICS4Wrapper, but hooks instead uses this `HooksICS4Wrapper` struct instead. 

I think they're functionally the same but would be good to have someone independently verify

Reply:
@asalzmann am I good to mark this resolved?

Reply:
Yes